components/aws/sagemaker/hyperparameter_tuning/component.yaml

```diff
@@ -150,7 +150,7 @@ outputs:
     description: 'The registry path of the Docker image that contains the training algorithm'
 implementation:
   container:
-    image: amazon/aws-sagemaker-kfp-components:0.5.0
+    image: akartsky/aws-kubeflow-sagemaker:latest
```

Comment:
What is the flow we are following here ? shouldn't the images be changed back before the PR is created ?

Reply:
need this for testing 
once it's approved will follow MCM